content/blog/2015-07_Meet Nukomeeter  Adam.md

```diff
@@ -0,0 +1,40 @@
+---
+created_at: 2015-07-02
+kind: article
+publish: true
+title: "Meet Nukomeeter: Adam"
+authors:
+- adam
+- kasia
+tags: interview
```

Comment:
I think it tag "interview" written in a wrong way. I've corrected it,
we'll see.

*Kasia Popek*

Communications Manager at Nukomeet


___


You can contact me at

mail *kasia@nukomeet.com*

mobile *+48 609 160 095*


Visit our website *www.nukomeet.com*




On Thu, Jul 9, 2015, at 09:17 PM, Tomek Kuklis wrote:
> In content/blog/2015-07_Meet Nukomeeter  Adam.md[1]:


> > @@ -0,0 +1,40 @@
> +--- +created_at: 2015-07-02 +kind: article +publish: true +title:
> "Meet Nukomeeter: Adam" +authors: +- adam +- kasia +tags: interview
> @myviridis[2]  I've got compilation error here , this is wrong format
> for tags. It looks like authors above


> — Reply to this email directly or view it on GitHub[3].


>



Links:

  1. https://github.com/nukomeet/nukomeet.com/pull/192#discussion_r34292475
  2. https://github.com/myviridis
  3. https://github.com/nukomeet/nukomeet.com/pull/192/files#r34292475